connector/profiler/README.md

```diff
@@ -39,7 +39,7 @@ For more information on async_profiler see the [Async Profiler Manual](https://k
 To enable code profiling, first enable the code profiling plugin via
 
 ```
-spark.plugins=org.apache.spark.executor.profiler.ExecutorProfilerPlugin
+spark.plugins=org.apache.spark.profiler.ProfilerPlugin
```

Comment:
Do we need to document anywhere that the name of the plugin and the related options have changed?

Reply:
`ExecutorProfilerPlugin` is new feature of Spark 4. So, it's okay.